index.markdown

```diff
@@ -355,12 +372,12 @@ redirect_from: '/moveit/'
                       ROS 2
                     </div>
                     <div class="release-versions__3">
-                      <span class="label label--blue">In Development</span>
+                      <span class="label label--green">RECOMMENDED</span>
```

Comment:
@L4co77 I find the bright green button with white text hard to read, can you make this text black or something?

Reply:
![image](https://user-images.githubusercontent.com/561060/124290273-77c08f00-db10-11eb-934b-18562a8f1a88.png)

Reply:
sure...u can just add class text-dark from [bootstrap ](https://getbootstrap.com/docs/4.0/utilities/colors/)